windows-cmd-extensions/data/commands.json

```
@@ -1,4 +1,40 @@
 [
+  {
+    "name": "AdvancedNetworkDiagnostics",
```

```suggestion
    "title": "AdvancedNetworkDiagnostics",
```
It should be `title` not `name`